data/json/items/magazine/50.json

```diff
@@ -14,7 +14,7 @@
     "type": "MAGAZINE",
     "name": "Barrett magazine",
     "//": "Fully loaded 10 magazine, presumably loaded with the 114.7 gram  M33 Ball cartridge, weighs 1868.801 grams. Source inetres.com",
-    "description": "Standard 10-round box magazine for a Barrett M107A1 anti-materiel rifle.",
+    "description": "Standard 10-round box magazine for a Barrett M107A1 anti-material rifle.",
```

Comment:
"Materiel" is actually correct: https://en.wikipedia.org/wiki/Anti-materiel_rifle

Reply:
Wow doing a search it seems there is not much of a standard in articles or even on some US government stuff. But its mostly spelled 'materiel' so your right. I'll change it back.